docs/en/test.md

```diff
@@ -35,12 +54,33 @@ To test pruning method, you can use following command
 python tools/${task}/test_${task}.py ${CONFIG_FILE} ${CHECKPOINT_PATH} --cfg-options algorithm.channel_cfg=${CHANNEL_CFG_PATH} [optional arguments]
 ```
 
+- `task`: one of ``mmcls``、``mmdet`` and ``mmseg``
 - `CHANNEL_CFG_PATH`: Path of `channel_cfg`. `channel_cfg` represents **config for channel of the subnet searched out**, used to specify different subnets for testing. An example for `channel_cfg` can be found [here](https://github.com/open-mmlab/mmrazor/blob/master/configs/pruning/autoslim/AUTOSLIM_MBV2_220M_OFFICIAL.yaml), and the usage can be found [here](https://github.com/open-mmlab/mmrazor/blob/master/configs/pruning/autoslim/README.md#test-a-subnet).
 
+For example,
+
+<pre>
+python ./tools/mmcls/test_mmcls.py \
+  configs/pruning/autoslim/autoslim_mbv2_subnet_8xb256_in1k.py \
+  <em>your_splitted_checkpoint_path</em> --metrics accuracy \
+  --cfg-options algorithm.channel_cfg=configs/pruning/autoslim/AUTOSLIM_MBV2_530M_OFFICIAL.yaml
+</pre>
+
 ## Distillation
 
 To test distillation method, you can use the following command
 
 ```bash
 python tools/${task}/test_${task}.py ${CONFIG_FILE} ${CHECKPOINT_PATH} [optional arguments]
 ```
+
+- `task`: one of ``mmcls``、``mmdet`` and ``mmseg``
+
+For example,
+
+<pre>
+python ./tools/mmseg/test_mmseg.py \
+  configs/distill/cwd/cwd_cls_head_pspnet_r101_d8_pspnet_r18_d8_512x1024_cityscapes_80k.py \
+  <em>your_splitted_checkpoint_path</em> \
```

Comment:
Distillation seems need not to split the checkpoint, so `your_checkpoint_path` may better than <del>`your_splitted_checkpoint_path`</del>